CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+|Version|version|**REQUIRED**| Provide the version number, or the release point of your dataset.|Provide the version number, or the release point of your dataset. Need to be a positive number. |
+|Licenses|licenses|**REQUIRED**| The use of license name abbreviations is suggested for specifying a license. Please visit [Creative Commons](https://creativecommons.org/share-your-work/) to choose the right licence for you.| The licences under which this dataset is shared.|
+|Keywords|keywords|**REQUIRED**| Tags associated with the dataset, which will help in its discovery. We suggest entering at least 5 keywords, different from the datatype.|Tags associated with the dataset, which will help in its discovery. These should be well known terms by the research community. |
+|Formats|distributions - format|**REQUIRED**| The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite)Primary data files format (*example*: csv, nifti, txt, fasta).| The technical format of the dataset distribution. Use the file extension or MIME type when possible. (Definition adapted from DataCite).|
+|Size|distributions - size|**REQUIRED**| Total size of the dataset.| The quantity of space required on disk (or other medium) for this dataset.|
+|Units|distributions - unit|**REQUIRED**| Unit in which the size is measured.(*example*: GB)|Unit in which the size is measured.(MB -> MegaByte, GB -> GiGaByte, TB -> TeraByte and PB -> PetaByte). |
+|Access |||The information about access modality for the dataset distribution.|The information about access modality for the dataset distribution.|
+|Landing Page|distributions - access - landingPage|**REQUIRED**| Web address (URL) where the original dataset can be found.|An URL (Web page) that contains information about the associated dataset or other research object and a direct link to the object itself. |
+|Authorizations|distributions - access - authorizations| **REQUIRED**|Types of verification that accessing the resource is allowed. Authorization occurs before successful authentication and refers to the process of obtaining approval to use a data set. Ideally specified from a controlled vocabulary or ontology.|This must be one of "Public", "Registered" or "Private". When this field is absent the value will be treated as "Public". Public: available to anyone, Registered: available to user authorized by the creator, Private: available only to the creator.|
+|Number of Files|extraProperties - files|**REQUIRED**| The number of files in the dataset.|Total number of files in the dataset. |
+|Number of Subjects|extraProperties - subjects|**REQUIRED**| The number of subjects in the dataset.|Total number of subjects constituting the dataset.|
+|CONP Status|extraProperties - CONP_status|**REQUIRED**| CONP Status.| CONP: created using funding from the CONP, Canadian: created in Canada without CONP funding, External: created outside of Canada.|
```

Comment:
```suggestion
|CONP Status|extraProperties - CONP_status|**REQUIRED**| CONP Status.| "CONP" = created using funding from the CONP; "Canadian" = created in Canada without CONP funding; "External" = created outside of Canada.|
```

I would tend to add a little explanation before to explain what the CONP status is. Not sure what that sentence would be though. Maybe @emmetaobrien can help with that?

Reply:
I will add that it is used to set the flag on the dataset list.